glacierscript.py

```diff
@@ -493,20 +493,35 @@ def write_and_verify_qr_code(name, filename, data):
     Write a QR code and then read it back to try and detect any tricksy malware tampering with it.
 
     name: <string> short description of the data
-    filename: <string> filename for storing the QR code
+    filename: <string> filename for storing the QR code. note that ".png" is added in function to make incrementing easier
     data: <string> the data to be encoded
     """
 
-    subprocess.call("qrencode -o {0} {1}".format(filename, data), shell=True)
+    # note: it would probably be better to write qr codes to RAM disk or secure volume. while shouldn't contain keys, may increase security by decreasing chance of unencrypted transaction history being discovered
+    QR_SUBDIR = "qrcodes"
+    QR_SUFFIX = ".png"
+    script_root = os.path.dirname(os.path.abspath(__file__))
+    QR_DIRPATH = script_root + "/" + QR_SUBDIR
```

Comment:
This moves the qr codes from their previous location, which means the Glacier doc needs to change to tell users where to find the file.

Reply:
While I think being able to move to different path might have future value (e.g. archiving the QRs in a secure volume in q-environment), to simplify at this point in time I've reverted the destination directory back to glacier root. Thanks!